### DISCUSSION TITLE
Accessing other extensions in settings

### DISCUSSION THREAD

**Discussion by Archie-2021** [brookhong/Surfingkeys#2019] [Q&A]
Is there a way to access other extensions in surfingkeys setting? Even if it is not directly possible is it possible to rebind the keys assigned to them to new keys (vim like keycords which are not an option in Chrome `settings > extensions > shortcuts` ?

**Comment by Janzulene** [comment]
I know vimium c can do this, they have a plugin called vimium-c-helpers:
https://www.sciencedirect.com/science/article/pii/S1046202318303827

But i am also wonder can it be achieved by just javascript in surfingkeys.

**Reply by Archie-2021** [reply]
Yeah, I have heard good things about vimium-c , have not tried it tho. It is too much for me to relearn all hotkeys + I like surfingkeys setup. It is going to be huge if it is possible to add such a feature.

**Reply by Janzulene** [reply]
yeah. The idea to use javascript setting your own shortcuts is so attractive.

**Reply by Janzulene** [reply]
Sadly, I tried vimium c helpers, but I found that it did not match what I wanted to do. It actually set the shortcut keys in the edge extension options, and then it could forward these keys to other plugins (in practice, it seemed to only work with vimium c, which allowed vimium c to have more shortcut keys), instead of letting vimium c use modes shortcut keys to control the switch function of other extensions.
